File: BAEKJOON/C/5_1_dimension_array/Prob_4344.c
#include <stdio.h>

int main()
{
    int N = 0;
    int sum = 0;
    int array = 0;
    int M = 0;
    float avg = 0.0;
    int num = 0;

    scanf("%d", &N);

    for(int i = 0; i<N; i++)
    {
        scanf("%d", &M);
        int array[M] = {0,};
        sum = 0;

        for(int j = 0; j<M; j++)
        {
            scanf("%d", &array[j]);
            sum = array[j] + sum;
        }
        avg = (float)(sum/(float)M);

        num = 0;
        for(int j = 0; j<M; j++)
        {
            if(avg<array[j])
            num = num + 1;
        }
        printf("%.3f%%\n", ((float)num/(float)M*100));

    }
}

File: BAEKJOON/Python/5_1_dimension_array/Prob_4344.py
N = int(input())
num = 0

for i in range(0,N,1):
    temp_array = list(map(int, input().split()))
    
    avg = (sum(temp_array)-temp_array[0]) / (len(temp_array)-1)

    for i in range(len(temp_array)):
        temp_array[i] = temp_array[i]/avg
        if (temp_array[i]>1):
            num = num + 1
    print(str("%.3f" %(100*num/(len(temp_array)-1)))+"%")
    num = 0
    avg = 0
    temp_array = []